engine/consensus/dkg/reactor_engine.go

```diff
@@ -312,7 +312,7 @@ func (e *ReactorEngine) handleEpochCommittedPhaseStarted(currentEpochCounter uin
 	if err != nil {
 		e.log.Fatal().Err(err).Msg("failed to set dkg")
 	}
-	log.Info().Msgf("successfully ended DKG, my beacon pub key for epoch %d is %x", nextEpochCounter, localPubKey.Encode())
+	log.Info().Msgf("successfully ended DKG, my beacon pub key for epoch %d is %s", nextEpochCounter, localPubKey.String())
```

Comment:
I guess it works this way too 
```suggestion
	log.Info().Msgf("successfully ended DKG, my beacon pub key for epoch %d is %s", nextEpochCounter, localPubKey)
```